<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskForm</class>
 <widget class="QWidget" name="TaskForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 216, 217);</string>
  </property>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="taskText">
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Текст Задания</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="TryButton">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Сдать</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="LoseButton">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Выйти</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="YourAnswerlabel">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Ваш ответ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="AnswerLineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
